subprojects/docs/src/docs/userguide/javaTesting.adoc

```diff
@@ -359,12 +400,13 @@ A common requirement for projects is to incorporate integration tests in one for
 The simplest way to add integration tests to your build is by taking these steps:
 
  1. Create a new source set for them
- 2. Add the required dependencies to the appropriate configurations for that source set
+ 2. Add the dependencies you need to the appropriate configurations for that source set
  3. Configure the compilation and runtime classpaths for that source set
+ 4. Create a task to run the integration tests
 
 You may also need to perform some additional configuration depending on what form the integration tests take. We will discuss those as we go.
 
-Here's a practical example that implements the above steps in a build script:
+Let's start with a practical example that implements the first three steps in a build script, centred around a new source set `intTest`:
```

Comment:
❓ would a user be likely to wonder what a source set is here?

Reply:
Not sure, but I can't believe no one called me out for that spelling of "centred" :smile:

Reply:
I'm adding a link to the first bullet point in the list above this paragraph. I think you're right that it's worth directing folks to information about what source sets are.